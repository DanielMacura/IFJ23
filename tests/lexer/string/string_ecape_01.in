let message = "Ahoj\n\"Sve'te \\\u{22}"