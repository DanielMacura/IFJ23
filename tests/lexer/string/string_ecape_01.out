KW_LET
VARIABLE
EQUALS
STRING:Ahoj\010"Sve'te\032\092"
EOF
